Makefile

```diff
@@ -89,6 +89,7 @@ build/init.sql: sql/privacy/init.sql
 clean:
 	rm -Rf build
 	find . -name 'Skargo.toml' -print0 | sed 's|Skargo.toml|target|g' | xargs -0 rm -rf
+	cd compiler && ${MAKE} clean
```

Comment:
I am not sure if we always want to clean the compiler, there are many times where the compiler doesn't change for long periods during development of the other parts of the system. The steps you have in the README don't need this change, right?

Also, elsewhere we usually use `$(MAKE)`, is there a reason to prefer `${MAKE}`?